client/js/components/map/publicdetail/controls/legendList/DpLayerLegend.vue

```diff
@@ -29,7 +29,7 @@
           <a
             :class="prefixClass('c-map__group-item display--block')"
             target="_blank"
-            :href="Routing.generate('core_file', { hash: planPdf.hash })"
+            :href="Routing.generate('core_file_procedure', { hash: planPdf.hash })"
```

Comment:
you need to add procedureId when using this route

Reply:
ja my bad, it's missing in other Vues too, i ll try to fix otherwise i ll search for Frontend help

Reply:
@mussbach all concerns should be handled and gefixed.